src/bentoml/_internal/runner/container.py

```diff
@@ -283,20 +283,32 @@ def to_payload(
                 # TODO: use fortan contiguous if it's faster
                 batch = np.ascontiguousarray(batch)
 
+            meta: dict[str, bool | int | float | str | list[int]] = {
```

Comment:
Forgot to ask this, can we annotate this with TypedDict?

Reply:
Then we need to change `create_payload` function's annotation. Let's do this in another pr